BNomad/View/PlaceInfoView/ReviewInfoCell.swift

```diff
@@ -12,40 +12,36 @@ class ReviewInfoCell: UICollectionViewCell {
     
     
     // MARK: - Properties
-    
-
-    // TODO: - 하드 코딩된 부분 전부 제거 --- 데이터 있는 부분 완료
-    
-    
-    var place: Place? {
-        didSet {
-            guard let place = place else { return }
-            mappingPlaceData(place)
-        }
-    }
 
+    static let cellIdentifier = "ReviewInfoCell"
+    let reviewInfoTitleLabel = UILabel()
+    let reviewCountLabel = UILabel()
+    let horizontalDivider1 = UILabel()
+    let horizontalDivider2 = UILabel()
+    let horizontalDivider3 = UILabel()
+    let horizontalDivider4 = UILabel()
+    let horizontalDivider5 = UILabel()
+    let reviewCollectionView: UICollectionView = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        let reviewCollectionView = UICollectionView(frame: .init(x: 0, y: 0, width: 100, height: 100), collectionViewLayout: flowLayout)
```

Comment:
```suggestion
        let reviewCollectionView = UICollectionView(frame: .zero, collectionViewLayout: flowLayout)
```
프레임을 오토레이아웃으로 만들어주기때문에 상단 frame에서는 `.zero`로 하는게 좋을 것 같아요!

Reply:
넵 참고하여 수정했습니다!